camera/camera_android.go

```diff
@@ -80,23 +82,18 @@ func (c *DroidCamera) NextImage() (img image.Image, err error) {
 		return nil, err
 	}
 
-	var yStride C.int
-	var yLen, cbLen, crLen C.int
-	var yPtr, cbPtr, crPtr *C.uint8_t
+	var jpegLen C.int
+	var jpegPtr *C.uint8_t
 
-	C.AImage_getPlaneRowStride(C.globalImage.image, 0, &yStride)
-	C.AImage_getPlaneData(C.globalImage.image, 0, &yPtr, &yLen)
-	C.AImage_getPlaneData(C.globalImage.image, 1, &cbPtr, &cbLen)
-	C.AImage_getPlaneData(C.globalImage.image, 2, &crPtr, &crLen)
+	C.AImage_getPlaneData(C.globalImage.image, 0, &jpegPtr, &jpegLen)
+	jpegData := C.GoBytes(unsafe.Pointer(jpegPtr), jpegLen)
 
-	c.img.YStride = int(yStride)
-	c.img.CStride = int(yStride) / 2
-
-	c.img.Y = C.GoBytes(unsafe.Pointer(yPtr), yLen)
-	c.img.Cb = C.GoBytes(unsafe.Pointer(cbPtr), cbLen)
-	c.img.Cr = C.GoBytes(unsafe.Pointer(crPtr), crLen)
+	img, err = jpeg.Decode(bytes.NewReader(jpegData))
```

Comment:
Any specific reason we aren't using the `DroidCamera` attribute `c.img` for the JPEG flow?

Reply:
Oh I see that that attribute is ycbr. Should we change that?

Reply:
Yeah good call! I guess we could use c.img but dont think we need to store img in the struct anymore. Removed it.